<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FTutor1D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="filteredPlot" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>450</width>
      <height>250</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QScrollBar" name="filteredHorizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>450</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="filteredVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>16</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="filteredLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>305</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtered signal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>303</y>
      <width>1000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="currentPlot" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>450</width>
      <height>250</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QScrollBar" name="currentHorizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>450</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="currentVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>16</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current frequency</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="magnitudeTab">
     <attribute name="title">
      <string>Magnitude</string>
     </attribute>
     <widget class="QScrollBar" name="magnitudeHorizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="magnitudeVerticalScrollBar">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>16</width>
        <height>251</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="magnitudePlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>250</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="phaseTab">
     <attribute name="title">
      <string>Phase</string>
     </attribute>
     <widget class="QScrollBar" name="phaseHorizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>450</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="phaseVerticalScrollBar">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>16</width>
        <height>250</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="phasePlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>250</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="originalPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>450</width>
      <height>250</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
   </widget>
   <widget class="QScrollBar" name="originalHorizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>450</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="originalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>305</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Signal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollBar" name="originalVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>16</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRevert_to_original"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEn_UK"/>
    <addaction name="actionCs_CZ"/>
    <addaction name="actionSk_SK"/>
    <addaction name="actionFr_FR"/>
    <addaction name="actionDe_DE"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionView_Help"/>
    <addaction name="separator"/>
    <addaction name="actionOfficial_Website"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuFilters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRevert_to_original">
   <property name="text">
    <string>Revert to original</string>
   </property>
  </action>
  <action name="actionEn_UK">
   <property name="text">
    <string>en_US</string>
   </property>
  </action>
  <action name="actionCs_CZ">
   <property name="text">
    <string>cs_CZ</string>
   </property>
  </action>
  <action name="actionSk_SK">
   <property name="text">
    <string>sk_SK</string>
   </property>
  </action>
  <action name="actionFr_FR">
   <property name="text">
    <string>fr_FR</string>
   </property>
  </action>
  <action name="actionDe_DE">
   <property name="text">
    <string>de_DE</string>
   </property>
  </action>
  <action name="actionView_Help">
   <property name="text">
    <string>View Help</string>
   </property>
  </action>
  <action name="actionOfficial_Website">
   <property name="text">
    <string>Official Website</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">QCustomPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
